api/src/main/resources/schema/workflow.json

```diff
@@ -36,7 +36,8 @@
       }
     },
     "dataInputSchema": {
-      "$ref": "datainputschema/datainputschema.json",
+      "type": "object",
```

Comment:
@tsurdilo can you check this. I cannot think of an alternative but Is this approach fine? this would lead us to maintain java SDK specific workflow schema

Reply:
i don't think that dataInputSchema in spec schema in spec has a type element: https://serverlessworkflow.io/schemas/0.8/workflow.json

so yeah idk where this comes from or why its needed?

Reply:
@tsurdilo just confirming, hope u looked at this part "existingJavaType": "io.serverlessworkflow.api.workflow.DataInputSchema"